packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -118,12 +118,13 @@ export class ComponentUtils {
   public async fetchAndSaveMetadataComponentProperties(
     metadataType: string,
     connection: Connection,
-    componentsPath: string
+    componentsPath: string,
+    folderName?: string
   ): Promise<string> {
-    const metadataQuery = {type: metadataType};
+    const metadataQuery = {folder: folderName, type: metadataType};
     const metadataFileProperties = await connection.metadata.list(metadataQuery);
     const result = {status: 0, result: metadataFileProperties};
-    const jsonResult = JSON.stringify(result);
+    const jsonResult = JSON.stringify(result, null, 2);
```

Comment:
Are the null and 2 needed here (and on line 140)?  I don't think there's anything that's requiting 2-space indentation of the JSON.

Reply:
@jeffb-sfdc I added `null` and `2` to pretty print json flile, which provides better readability.

Reply:
got it, thanks.  I don't think we normally pretify the JSON.  Let's keep it then.